app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -406,7 +410,10 @@ class MainPageActivity : AppCompatActivity(), OnRequestPermissionsResultCallback
     fun goToCprActivity(view : View) {
         goToActivity(CprRateActivity::class.java)
     }
-
+    /** Called when the user taps the forum button */
```

Comment:
You can do a switch case to allow not needing a gazillion goToXActivity

Reply:
Yea, it was mentioned in the code review. It should be done by Alex now